tokio/src/runtime/blocking.rs

```diff
@@ -0,0 +1,40 @@
+pub(crate) use self::variant::*;
+
+#[cfg(feature = "blocking")]
+mod variant {
```

Comment:
Handles compilation compatibility when feature `blocking` is enabled / disabled.

Reply:
could be worth adding a comment explaining what's going on here (and also, what the difference between this module and the other `blocking` mods is?